docs/API/Elements.md

```diff
@@ -253,6 +279,15 @@ Any element property can be used to filter by using the <code>@</code> operator.
 var big = a.findOne('..(@size==10)');
 ```
 
+Here is an example of searching for a child with a name set in the inspector
+
+```js
+const child = '{[Find this child:string]}';
```

Comment:
I would add just a little more context here to explain that the `'{[Inspector variable label : variable type]}'` is required to make a variable accessible from the inspector.  But if you don't need a variable to be accessible from the inspector, you can declare a variable in the normal way too.